WordPress/Classes/ViewRelated/Post/Geolocation/PostGeolocationView.m

```diff
@@ -77,25 +77,31 @@ - (void)setAddress:(NSString *)address
 
 - (void)setCoordinate:(Coordinate *)coordinate
 {
+    MKCoordinateRegion defaultRegion = MKCoordinateRegionMakeWithDistance(coordinate.coordinate, 200, 100);
```

Comment:
Maybe explicit decimal values for type consistency since CLLocationDistance is a typealias for double.